mlrun/db/base.py

```diff
@@ -91,7 +91,9 @@ def del_runs(self, name="", project="", labels=None, state="", days_ago=0):
         pass
 
     @abstractmethod
-    def store_artifact(self, key, artifact, uid, iter=None, tag="", project=""):
+    def store_artifact(
+        self, key, artifact, uid, iter=None, tag="", project="", calc_size=False
+    ):
```

Comment:
why do we need to take that argument? this granularity is too wide
calc size should happen at best effort anyway